packages/babel-helper-create-class-features-plugin/src/misc.js

```diff
@@ -84,12 +82,17 @@ export function injectInitialization(path, constructor, nodes, renamer) {
 
 export function extractComputedKeys(ref, path, computedPaths, file) {
   const declarations = [];
-
+  const state = {
+    classBinding: path.node.id && path.scope.getBinding(path.node.id.name),
+    file,
+  };
   for (const computedPath of computedPaths) {
-    computedPath.traverse(classFieldDefinitionEvaluationTDZVisitor, {
-      classBinding: path.node.id && path.scope.getBinding(path.node.id.name),
-      file,
-    });
+    if (computedPath.get("key").isReferencedIdentifier()) {
+      handleClassTDZ(computedPath.get("key"), state);
+    }
+    computedPath
```

Comment:
Nit: we can use `else` here, since an identifier doesn't have any child anyway.

Reply:
Done. Please review :)